service-api/src/main/java/greencity/dto/HasPasswordDto.java

```diff
@@ -0,0 +1,14 @@
+package greencity.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class HasPasswordDto {
```

Comment:
Was it necessary to create a separate dto? Instead of creating a new Boolean variable

Reply:
Yes, because we get the same dto from GreenCityUser